statshouse.hpp

```diff
@@ -887,19 +898,35 @@ class Registry {
 		}
 		uint64_t seed{static_cast<uint64_t>(std::random_device{}())};
 	};
+	struct bucket;
 	struct multivalue {
-		void write(multivalue_view &src, Registry *r, PRNG *rand) {
+		void write(multivalue_view &src, Registry *r, bucket *b) {
 			if (src.values) {
-				write(values, src.values, src.values_count, r, rand);
+				if (b->last != 0) {
```

Comment:
I still think that it's cursed way of handling modes. Can we at least throw something if `b->last && b->regular`?